client/webserver/site/src/js/register.js

```diff
@@ -161,7 +140,7 @@ export default class RegistrationPage extends BasePage {
     this.auth()
```

Comment:
A couple lines up there is `if (seed) this.walletForm.setRegMsg('Your Decred wallet is required')`.
I'm struggling to recall why restoring from seed means the Decred wallet is required.
@buck54321

Reply:
It would make sense if wallet was required when not restoring from seed, because pay fee.

Reply:
Well in any case, seems like this conditional message is no longer correct.